Makefile

```diff
@@ -62,6 +62,11 @@ deploy-k8s-env: pre-deploy-crds postgresql-secret-on-k8s ## Deploy all controlle
 undeploy-k8s-env: pre-undeploy-crds ## Remove all controller/DB workloads from K8s.
 	$(KUSTOMIZE) build $(MAKEFILE_ROOT)/manifests/overlays/k8s-env | kubectl delete -f - 
 
+deploy-k8s-env-e2e: kustomize postgresql-secret-on-k8s ## Deploy all controller/DB workloads to K8s for e2e tests, use e.g. IMG=quay.io/pgeorgia/gitops-service:latest to specify a specific image
```

Comment:
Just need to add `pre-deploy-crds` or `pre-undeploy-crds` to these 2 Makefile targets, as we are doing in the previous target in the Makefile (these were very recently added by another PR)

Reply:
Done